course_content/iris_course/3.Subcube_Extraction.ipynb

```diff
@@ -279,12 +283,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def less_than_10(cell):\n",
-    "    \"\"\"Return True for values that are less than 10.\"\"\"\n",
-    "    return cell < 10\n",
+    "def near_357p5(cell):\n",
```

Comment:
Should this not be `near_358p5`?

Reply:
On second thoughts, and offline discussion, it's better to *not* discuss equality testing here at all.